client/my-sites/site-settings/disconnect-site/another-reason.jsx

```diff
@@ -0,0 +1,55 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormTextArea from 'components/forms/form-textarea';
```

Comment:
Nit: should it be `FormTextarea` (with a lowercase `a`) instead?

Reply:
Ah yeah, right, that's how it's imported elsewhere.